spec/support/vcr_cassettes/hca/health_check_downtime.yml

```diff
@@ -9,36 +9,56 @@ http_interactions:
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ns1="http://va.gov/schema/esr/voa/v1"
         xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://va.gov/service/esr/voa/v1"><soap:Body><ns1:retrieveFormSubmissionStatusRequest><ns1:formSubmissionId>377609264</ns1:formSubmissionId></ns1:retrieveFormSubmissionStatusRequest></soap:Body></soap:Envelope>
     headers:
+      Accept:
+      - text/xml;charset=UTF-8
+      Content-Type:
+      - text/xml;charset=UTF-8
       User-Agent:
-      - Faraday v0.9.2
+      - Vets.gov Agent
       Date:
-      - Mon, 12 Dec 2016 14:06:07 GMT
+      - Tue, 20 Aug 2024 16:40:26 GMT
       Content-Length:
       - '472'
-      Content-Type:
-      - text/xml;charset=UTF-8
       Accept-Encoding:
       - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
-      Accept:
-      - "*/*"
   response:
     status:
-      code: 302
+      code: 200
```

Comment:
This is the only one that produced a confusing result. Why is it successful now, but returned a 302/NOT FOUND before? Perhaps this was manually done to suggest an outage? Maybe they worked with the upstream service to "create" an outage for recording purposes? Maybe we need to use a different `formSubmissionId`?